mta/tests/test_login.py

```diff
@@ -4,8 +4,9 @@
 
 
 @pytest.mark.smoke
-@pytest.mark.web_console
-def test_login(application):
+@pytest.mark.parametrize("mta_context", ["ViaWebUI", "ViaOperatorUI"])
+def test_login(application, mta_context):
     """Test login nav destination"""
+    application.mta_context = mta_context
```

Comment:
so no need of this, if we parametrized like other test.

Reply:
Done